precompile/modules/initia_stdlib/sources/dex.move

```diff
@@ -27,6 +27,20 @@ module initia_std::dex {
         coin_b_store: Object<FungibleStore>
     }
 
+    struct FlashSwap has key {
```

Comment:
we probably dont need this anymore?

Reply:
No that is used to keep the borrowed amount so the pool_amounts still need to use that to fetch pool reserve. And also that is used to lock the pool to prevent recursive flash swal

Reply:
+ we don't want to change Pool object at this time because it would break the compatibility on `initiation-2`

Reply:
Seems reasonable